ipynb/examples/utils/testenv_example.ipynb

```diff
@@ -1,43 +1,126 @@
 {
  "cells": [
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test environment API - TestEnv\n",
+    "\n",
+    "The test environment is primarely defined by the target configuration (see <b>conf</b> below).\n",
```

Comment:
Will do! I put the html elements as I think I've seen it before done like that and it worked :). Although your way looks nicer.